### PR TITLE
(CDAP-9444) Update TMS metrics doc.

### PR DESCRIPTION
**Pull request by chtyim**

### CODE REVIEW

File: cdap-docs/reference-manual/source/http-restful-api/metrics.rst [hunk 1]
```diff
@@ -355,6 +355,39 @@ These metrics are available for the CDAP transaction service:
...
+   * - ``system.persist.queue.size``
+     - Number of messages in the queue that are persisted in one batch
+   * - ``system.cache.add.requests``
+     - Number of entries requested to add to the messaging cache
+   * - ``system.cache.entries.added``
+     - Number of entries added to the messaging cache
+   * - ``system.cache.entries.removed``
+     - Number of entries removed from the messaging cache
+   * - ``system.cache.add.reduce.weight``
+     - Number of times that the cache reduce weight logic executed while adding entries to the cache.
```
Comment (sreevatsanraman) [inline cdap-docs/reference-manual/source/http-restful-api/metrics.rst]: ```weight logic is executed```

File: cdap-docs/reference-manual/source/http-restful-api/metrics.rst [hunk 2]
```diff
@@ -355,6 +355,39 @@ These metrics are available for the CDAP transaction service:
...
+       This number ideally should be very small for the cache to have good performance.
+   * - ``system.cache.scan.reduce.weight``
+     - Number of times that the cache reduce weight logic executed while scanning the cache.
```
Comment (sreevatsanraman) [inline cdap-docs/reference-manual/source/http-restful-api/metrics.rst]: ```reduce weight logic is executed ```

**Comment by sreevatsanraman** [comment]
Two minor comments. Fix and merge.

File: cdap-docs/reference-manual/source/http-restful-api/metrics.rst (hunk 2 above)
Comment (chtyim) [reply cdap-docs/reference-manual/source/http-restful-api/metrics.rst]: is or was?

**Comment by chtyim** [comment]
Only doc change. Merging.

**Pull request merged**